package.json

```diff
@@ -10,15 +10,15 @@
     "comma-number": "^2.0.1",
     "discord.js": "^12.5.1",
     "dotenv": "^8.2.0",
-    "minecraft-server-util": "^3.4.2",
+    "minecraft-server-util": "^3.4.3",
     "mysql": "^2.18.1",
     "ora": "^5.3.0",
     "twitter": "^1.7.1"
   },
   "devDependencies": {
-    "eslint": "^7.17.0",
+    "eslint": "^7.22.0",
     "inquirer": "^8.0.0",
-    "npm": "^7.4.1"
+    "npm": "^7.6.3"
```

Comment:
You seem to have updated some dependencies here! Although that's nice of you to do, we use Dependabot for this, so keep that in mind in the future.